/n/home13/chloe00/miniforge3/envs/eco/lib/python3.12/site-packages/jax/_src/ops/scatter.py:92: FutureWarning: scatter inputs have incompatible types: cannot safely cast value from dtype=int64 to dtype=int32 with jax_numpy_dtype_promotion='standard'. In future JAX releases this will result in an error.
  warnings.warn(
Population: 32, Food: 29
Traceback (most recent call last):
  File "/n/holylfs06/LABS/sham_lab/Users/chloe00/dirt/dirt/examples/nomnom/train_nomnom.py", line 130, in <module>
    train(key, params)
  File "/n/holylfs06/LABS/sham_lab/Users/chloe00/dirt/dirt/examples/nomnom/train_nomnom.py", line 97, in train
    train_state, active_players, logging_info = train_epoch(
                                                ^^^^^^^^^^^^
  File "/n/holylfs06/LABS/sham_lab/Users/chloe00/dirt/dirt/examples/nomnom/train_nomnom.py", line 76, in train_epoch
    train_state_active_players, trajectories = jax.lax.scan(
                                               ^^^^^^^^^^^^^
  File "/n/holylfs06/LABS/sham_lab/Users/chloe00/dirt/dirt/examples/nomnom/train_nomnom.py", line 68, in scan_body
    train_state, _ = train_state_active
    ^^^^^^^^^^^^^^
ValueError: too many values to unpack (expected 2)
--------------------
For simplicity, JAX has removed its internal frames from the traceback of the following exception. Set JAX_TRACEBACK_FILTERING=off to include these.
